apps/elf-council-frontend/package.json

```diff
@@ -4,7 +4,6 @@
     "@babel/types": "^7.17.0",
     "@blueprintjs/popover2": "^1.0.0",
     "@elementfi/base": "^1.0.0",
-    "@elementfi/elf-council-proposals": "^1.0.2",
```

Comment:
We'll need to add the `@elementfi/council-proposals` here and run `yarn` to rebuild yarn.lock